src/ucp/rma/flush.c

```diff
@@ -481,17 +484,19 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_ep_flush, (ep), ucp_ep_h ep)
 
 UCS_PROFILE_FUNC(ucs_status_t, ucp_worker_fence, (worker), ucp_worker_h worker)
 {
-    unsigned rsc_index;
+    ucp_rsc_index_t rsc_index;
+    ucp_worker_iface_t *wiface;
     ucs_status_t status;
 
     UCP_WORKER_THREAD_CS_ENTER_CONDITIONAL(worker);
 
-    for (rsc_index = 0; rsc_index < worker->context->num_tls; ++rsc_index) {
-        if (worker->ifaces[rsc_index].iface == NULL) {
+    ucs_for_each_bit(rsc_index, worker->context->tl_bitmap) {
```

Comment:
why do we need ucs_for_each_bit here? can't we just go over the array consecutively (without knowing rsc_index)?
(this comment can be relevant to other for loops which were converted to ucs_for_each_bit...)